Octokit/Http/IDeferredRequest.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Octokit
+{
+    public class ApiOptions
+    {
+        public int? StartPage { get; set; }
+        public int? PageCount { get; set; }
+        public int? PageSize { get; set; }
+        public string Accepts { get; set; }
```

Comment:
This might also be how we support different media types in the response without hard-coding :shit: everywhere. Added a task to explore this further.

cc @ammeep @thedillonb